storage_scrubber/src/main.rs

```diff
@@ -101,6 +106,12 @@ enum Command {
 async fn main() -> anyhow::Result<()> {
     let cli = Cli::parse();
 
+    tracing::info!(
+        "neon-storage-scrubber version: {}, build_tag {}",
```

Comment:
nit: maybe remove the `neon-storage-scrubber ` prefix?  We will know from log tags which service this is.

(also I have a muscle memory for searching logs for `INFO version` :-))